src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -316,10 +314,7 @@
       (when-not disable-update-checking
         (maybe-check-for-updates product-name update-server read-db))
 
-      (let [app (->> (server/build-app globals)
-                     (compojure/context url-prefix []))]
-        (log/info "Starting query server")
-        (add-ring-handler service app))
+      (log/info "Starting query server")
```

Comment:
Good catch, I'm removing this one and putting a starting and finished starting message in pdb_routing